mountpoint-s3/src/logging/event_log.rs

```diff
@@ -0,0 +1,35 @@
+use crate::fs::error_metadata::{MOUNTPOINT_ERROR_INTERNAL, MOUNTPOINT_ERROR_UNSUPPORTED};
```

Comment:
Would it make sense to briefly describe the event log here? And also add some comments on the 2 functions?